File: OOPRecords.Model/AppConfig.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;

namespace OOPRecords.Model
{
    public class AppConfig
    {
        public static Type[] Services()
        {
            return new[] {
                    typeof(StudentRepository),
                    typeof(TeacherRepository),
                    typeof(SubjectRepository),
                    typeof(SetRepository),
                };
        }

        public static IDictionary<string, Type> MainMenus()
        {
            return new Dictionary<string, Type>()
            {
                ["Students"] = typeof(StudentRepository),
                ["Staff"] = typeof(TeacherRepository),
                ["Subjects"] = typeof(SubjectRepository),
                ["Sets"] = typeof(SetRepository),
            };
        }

        public static DbContext CreateDbContext()
        {
            return new DatabaseContext("OOPRecords");
        }
    }
}

File: OOPRecords.ModelVB/AppConfig.vb
Imports System.Data.Entity

Public Module AppConfig
    Function Services() As Type()
        Return {
            GetType(StudentRepository),
            GetType(TeacherRepository),
            GetType(SubjectRepository),
            GetType(SetRepository)
            }
    End Function

    Function MainMenus() As IDictionary(Of String, Type)
        Return New Dictionary(Of String, Type)() From
            {
                {"Students", GetType(StudentRepository)},
                {"Staff", GetType(TeacherRepository)},
                {"Subjects", GetType(SubjectRepository)},
                {"Sets", GetType(SetRepository)}
            }
    End Function

    Function CreateDbContext() As DbContext
        Return New DatabaseContext("OOPRecords")
    End Function
End Module